functions/go/policy-controller-validate/README.md

```diff
@@ -0,0 +1,126 @@
+# policy-controller-validate
+
+### Overview
+
+<!--mdtogo:Short-->
+
+Validate the KRM resources using the policy controller.
+
+<!--mdtogo-->
+
+
+config-root/.../*-constraint.yaml and *-template.yaml define Policy Controller constraints and templates which all configs in config-root/ must pass.
+
+### Synopsis
+
+<!--mdtogo:Long-->
+You can use the policy controller to validate KRM resources. To learn more about
+the policy controller, see: https://cloud.google.com/anthos-config-management/docs/concepts/policy-controller.
+
+The function takes 3 types of resources from the input resource list:
+
+- constraint templates
+- constraints
+- KRM resources to be audited against
```

Comment:
I'm not sure if 'audit' is the established terminology here (Refer to Gatekeeper OSS docs).

Common phrasing: Constraint policies is enforced on KRM resources.